repos/system_upgrade/el7toel8/actors/dhclientreport/actor.py

```diff
@@ -0,0 +1,33 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_with_remediation
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+class CheckDhclientActor(Actor):
+    """
+    Check if dhclient is installed and report backward incompatible change in CLI options
+    """
+
+    name = "dhclient"
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+
+    def is_dhclient_installed():
+        for fact in self.consume(InstalledRedHatSignedRPM):
+            for rpm in fact.items:
+                if rpm.name == 'dhclient':
+                    return True
+        return False
+    
+    def process(self):
+        if is_dhclient_installed():
+            title = 'dhclient command line options changed'
+            summary = 'dhclient command line options have been changed ' \
```

Comment:
Nit: I'd use multiline strings instead \, something like 
summary = ("dhclient command line options have been changed "
                   "since Rhel 7.7 release")

Reply:
I started with this format actually but checked other actors and chose style they use.